src/org/scada_lts/service/model/MultiChangeHistoryComponentDTO.java

```diff
@@ -0,0 +1,33 @@
+package org.scada_lts.service.model;
+
+import java.util.Set;
+
+/**
+ * @author  grzegorz.bylica@abilit.eu on 17.10.2019
+ */
+public class MultiChangeHistoryComponentDTO {
+
+    private String xIdViewAndComponent;
+    private Set<MultiChangeHistoryDTO> history;
```

Comment:
Is it possible to change names "history" to more understandable . It means that multiChangeHistoriesDTO. Histories -because we have many instances of history.

Reply:
I'd rather not change the name